src/mechglue/gss_plugin.c

```diff
@@ -65,6 +65,8 @@ enum gpp_behavior gpp_get_behavior(void)
     return behavior;
 }
 
+static void gpp_init_special_available_mechs(const gss_OID_set mechs);
```

Comment:
Instead of forward-declaring, why not just put the function up here instead, since it's new code?

Reply:
want to keep it close to the other similar functions